TraitsExpanded/TraitSets/BigCharacterTraitSet.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+using TaleWorlds.MountAndBlade.Diamond;
+using TraitsExpanded.Traits;
+
+namespace TraitsExpanded.TraitSets
+{
+    public class BigCharacterTraitSet : TraitSet
+    {
+        public override string ID { get; set; } = "BigCharacterTraitSet";
```

Comment:
how about for trait sets we have a friendly name and an Id? for uniqueness the ID can be a guid, and the friendly name can be what we use when displaying it ingame

Reply:
Agreed